controller/cache/cache.go

```diff
@@ -149,6 +150,8 @@ type ResourceInfo struct {
 	PodInfo *PodInfo
 	// NodeInfo is available for nodes only
 	NodeInfo *NodeInfo
+
+	manifestHash *string
```

Comment:
Why a pointer? Would an empty string suffice for the "does not exist" case?

Reply:
No specific reasons. I will switch to a string.